src/constants/Socket.js

```diff
@@ -0,0 +1,11 @@
+export const SocketConst = {
+  getPost: 'get_post',
```

Comment:
all const names **must** be uppercase for example `GET_POST:  'get_post'`, **SocketConst** the same